src/admin/collectionsOrBundles/views/CollectionsOrBundlesOverview.tsx

```diff
@@ -509,10 +509,10 @@ const CollectionsOrBundlesOverview: FunctionComponent<CollectionsOrBundlesOvervi
 			return;
 		}
 
-		const detailRoute = isCollection
-			? APP_PATH.COLLECTION_EDIT.route
-			: APP_PATH.BUNDLE_EDIT.route;
-		const link = buildLink(detailRoute, { id });
+		const link = buildLink(
+			isCollection ? APP_PATH.COLLECTION_EDIT.route : APP_PATH.BUNDLE_EDIT.route,
+			{ id }
```

Comment:
It's best to use `APP_PATH.COLLECTION_EDIT_TAB.route` and add a param of `{ id, tabId: CollectionCreateUpdateTab.CONTENT }` in order for the first tab to be selected otherwise this will cause unwanted behaviour for other components.